components/SpotlightShare/icon.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Icon = styled.img`
+  float: left;
+  margin: 0 5px;
+  height: 3em;
+  width: 3em;
+`;
+
+const SVG = (props: any) => (
+  <div>
+    <a href="#">
```

Comment:
I expect this to have a functioning `href` ... add it as a prop. Add an `onClick` prop as well.